src/main/webapp/chart.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Charts Project(Part 1)</title>
+    <!-- take in core charts package -->
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+    <script type="text/javascript">
+      google.charts.load('current', {packages: ['corechart']});
```

Comment:
This script needs to live in its own file, similar to what is done for user-page. Put the code into src/webapp/js/charts-page-loader.js and include it here. Again, see user-page.html for the example of how to do this.

Reply:
Put the script in a separate javascript file